### PR TITLE
Quality of Life Update for Detective

### PR DESCRIPTION
**Pull request by ArbitraryKnapsack**
## About the PR 
-Created a brand new radio channel just for the detective [135.1 or :d]

-Spy Stickers now broadcast on :d instead of :g!  No more annoying the entire security department!
-Detective gets a special Detective's Headset that listens to both :d and :g
-Added a little magnifying glass icon that differentiates the Detective from the rest of the Security department

## Why's this needed? 
Currently Spy Stickers are a thematically awesome piece of equipment for detectives that are rendered useless by broadcasting on :g.  Now the detective can use them to setup secret comms, spy on the station, or do whatever other investigative things he wants without ruining security's ability to communicate!

The special icon for detective also gives everyone an easy way to identify the detective, especially when things have gotten bad on the station.  Now everyone can identify the detective!  This is not just cool, it also helps security communicate better when they know who is who.

## Changelog

```changelog
(u)ArbitraryKnapsack:
(*) Created a new radio channel, :d, for the detective's private use.  Keep tabs on the station without annoying all of security.
(+) Made a little chat icon for detective to visually identify them at a glance.
```

### CODE REVIEW

**Review by flrsh — CHANGES_REQUESTED** [review]
Nice QoL, though the random chatters over security will be missed (by me at least)!

I would prefer a non-headset piece of equipment that listened on the spy sticker frequency rather than creating a whole new radio channel just for the detective. Additionally, I don't see the new chat icon. In general, I think it'd be best if chat icons were easy to discern at a glance (typically this means text instead of symbols), and weren't too granular.

**Comment by ArbitraryKnapsack** [comment]
> Nice QoL, though the random chatters over security will be missed (by me at least)!
> 
> I would prefer a non-headset piece of equipment that listened on the spy sticker frequency rather than creating a whole new radio channel just for the detective. Additionally, I don't see the new chat icon. In general, I think it'd be best if chat icons were easy to discern at a glance (typically this means text instead of symbols), and weren't too granular.

The new chat icon is in browserassets/radio_icons/det.png on my branch.  Not sure why it isn't working or if new file adds don't work through github--I'm not super familiar with the whole thing.  If there's a step missing to add that to the PR, let me know, but I'm embedding it here for reference:
[https://i.imgur.com/xokYH0q.png](url)
I'm also not a spriter, but if the little magnifying glass is too annoying I can try to make one just read DET instead.  Conversations on the discord made it sound like people liked the idea of a magnifying glass and I somehow managed to make one while staring at radio code.  

I did consider a non-headset piece of equipment, but at the end of the day ran into a few issues with that:

1. The detective who is already used to the stickers outputting into their headset
2. Adding even more gear to the already crowded gearset of the detective would get confusing.  
3. I'm by no means expert on goon radio code, so finding a way to do this without also broadcasting to everyone in the vicinity is beyond my very limited code capabilities.  If I understand correctly, doing such a thing would stop the 'spy' part of the spy stickers entirely as it'd just yell to everyone around.  Also making it broadcast on another frequency already works pretty much the same as having a separate radio channel as far as code handling goes.  They only real difference I can think of would be the nuisance of how to listen to it or being able to speak through your stickers remotely.

**Review by pali6 — APPROVED** [review]

File: _std/defines/radio.dm [hunk 1]
```diff
@@ -2,7 +2,8 @@
 #define RADIOC_STANDARD "#008000"
 #define RADIOC_INTERCOM "#006080"
 #define RADIOC_COMMAND "#334E6D"
-#define RADIOC_SECURITY "#E00000"
+#define RADIOC_SECURITY "#e00000"
```
Comment (ZeWaka) [inline _std/defines/radio.dm:5]:
```suggestion
#define RADIOC_SECURITY "#E00000"
```

**Review by ZeWaka — APPROVED** [review]

**Comment by flrsh** [comment]
Whoops I forgot about this. We had discussed this in Discord and I think @ArbitraryKnapsack had made some additional changes?

**Review by Adharainspace — COMMENTED** [review]
the standard library edits seem good to me. unsure about the pr as a whole though. someone else weigh in?

**Review by pali6 — APPROVED** [review]

**Pull request merged**